src/elm/Pages/Build/History.elm

```diff
@@ -66,13 +52,36 @@ view now timezone page limit rm =
             text ""
 
 
+viewDeploymentBuilds : Posix -> Zone -> Org -> Repo -> Builds -> Html msg
+viewDeploymentBuilds now timezone org repo builds =
+    let
+        page : Page
+        page =
+            NotFound
```

Comment:
perhaps i misunderstand. are we sending users to a 404?

Reply:
IIRC, we don't have access to the page in the element we are using this, NotFound is just a placeholder, and afaict it doesn't do anything other than knowing what page the user was coming from.